src/include/stir/SeparableGaussianImageFilter.h

```diff
@@ -74,12 +72,19 @@ class SeparableGaussianImageFilter :
   //! Default constructor
   SeparableGaussianImageFilter();
 
+  void set_fwhms(const BasicCoordinate< num_dimensions,float>&);
+  void set_max_kernel_sizes(const BasicCoordinate< num_dimensions,int>&);
+  void set_normalise(const bool);
+private:
```

Comment:
this line needs to be moved down after the `get_` functions (but before the declaration of `fwhms` member). Otherwise we won't see these members from Python etc.